### PR TITLE
Switch to the new version of jacogr/polkadot-js-tools

### LINKED ISSUE #1230: Upgrade to latest polkadot-js/tools
Per @rlaferla, the latest version is https://github.com/polkadot-js/tools/releases/tag/v0.53.5 and contains changes that @wilwade made.

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to upgrade the polkadotjs tools for metadata comparison.

Closes #1230 

This also includes PR #1601 for expediency. So also part of #1362

Ended up needing to run each one in a docker container with a new docker network. This allowed everything to talk correctly.

# Checklist
- [x] Tested with: make start + docker run --rm --platform=linux/amd64 --pull always jacogr/polkadot-js-tools:0.55.3 metadata wss://1.rpc.frequency.xyz ws://host.docker.internal:9944
- [x] Tested with test release Running: https://github.com/LibertyDSNP/frequency/actions/runs/5282806513

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -153,7 +155,8 @@ jobs:
       matrix:
         # This job intermittently fails on EKS runners and must be run on standalone until
         # https://www.pivotaltracker.com/story/show/185045683 is resolved.
-        os: [[self-hosted, Linux, X64, build]]
+        # os: [[self-hosted, Linux, X64, testing, v2], [self-hosted, Linux, ARM64,  testing, v2]]
+        os: [[self-hosted, Linux, X64, testing, v2]]
```
Comment (rlaferla) [inline .github/workflows/release.yml:159]: What is the purpose of "testing" and "v2"?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -562,8 +579,9 @@ jobs:
           echo "Test version: $BIN_VERSION" >> ${{env.OUTPUT_FILENAME}}
           echo "----------------------------------------------------------------------" >> ${{env.OUTPUT_FILENAME}}
       - name: Compare Metadata
+        timeout-minutes: 10
         run: |
-          CMD="docker run --pull always --network host frequencychain/polkadot-js-tools:v0.52.29 metadata ws://localhost:9946 ws://localhost:9944"
+          CMD="docker run --pull always --net=net-${{env.RELEASE_BRANCH_NAME}} jacogr/polkadot-js-tools:0.55.3 metadata ws://ref-node:9944 ws://test-node:9944"
```
Comment (rlaferla) [inline .github/workflows/release.yml:584]: If you specify "--pull always" to get the latest version, do you still need to specify 0.55.3?

Comment (mattheworris) [reply .github/workflows/release.yml:584]: `--pull always` doesn't affect the version that gets pulled. It specifies to always pull the image regardless of whether or not the image is already cached locally.

**Review by mattheworris — APPROVED** [review]
Makes sense to me 🚀

File: .github/workflows/release.yml (hunk 1 above)
Comment (wilwade) [reply .github/workflows/release.yml:159]: These are the new eks runners.

**Pull request merged**